CenterForOpenScience/osf.io
# [WIP] fixes for auth addon for BB/GH/GL

## Purpose

Fixing auth errors for Bitbucket / Github / Gitlab so that they don't break OSF project loading.

## Changes

* Make BB/GH/GL's NodeSetting subclasses more error-tolerant.
* Fix assumptions in GL/BB fangorn customizations so that they don't break addon loading.

## QA Notes

*TBD*

> Please make verification statements inspired by your code and what your code touches.
> - Verify
> - Verify
> 
> What are the areas of risk?
> 
> Any concerns/considerations/questions that development raised?

## Documentation

No documentation should need to be updated, this is just better error handling.

## Side Effects

None expected.

## Ticket

Nope!
